src/Raven.Studio/typescript/components/pages/database/tasks/shared/OngoingTaskOperationConfirm.tsx

```diff
@@ -105,6 +105,15 @@ export default function OngoingTaskOperationConfirm(props: OngoingTaskOperationC
                         {idx < taskGroups.length - 1 && <hr className="m-0" />}
                     </div>
                 ))}
+
+                {type === "disable" && (
```

Comment:
Will this warning appear if we disable the `Subscription` ongoing task?

Reply:
As I see the situation, if we want to be as precise as possible in our warnings, we need to handle four different states:

1. When only one `Subscription` task is selected (no warning needed):

```
// no warning message to be shown
```

2. When only one non-`Subscription` task is selected, and a warning is needed:

```
Please note, <strong>disabling</strong> this task will lead to continuous tombstone accumulation until it is re-enabled or deleted, resulting in increased disk space usage.
```

3. When multiple tasks are selected, and the warning applies to all:

```
Please note, <strong>disabling</strong> these tasks will cause continuous tombstone accumulation until they are re-enabled or deleted, leading to an increase in disk space usage.
```

4. When multiple tasks are selected, and one of them is a `Subscription`:

```
Please note, <strong>disabling</strong> the selected tasks (excluding <strong>Subscription</strong> tasks) will lead to continuous tombstone accumulation until they are re-enabled or deleted, which will increase disk space usage. This warning does not apply to Subscription tasks.
```

Is this feasible?

Reply:
Done https://github.com/ravendb/ravendb/pull/16960/commits/c88d630c462e0be36f02e6fb7bbd5dfe343fec66

Reply:
Yeah, looks good now!

Two `subscription` tasks and `backup` task to disable:
![image](https://github.com/ravendb/ravendb/assets/7088748/d91cb54c-d34d-4a69-a4ba-3320d3b20a06)

`Backup` task and `RavenDB ETL` task to disable:
![image](https://github.com/ravendb/ravendb/assets/7088748/2d6bf576-50f1-4727-a71c-ba24b4e943c7)

Only `backup` task to disable:
![image](https://github.com/ravendb/ravendb/assets/7088748/911d21ab-0971-4a2d-9757-fd2a9b476ff1)

Only `subscription` task to disable:
![image](https://github.com/ravendb/ravendb/assets/7088748/c8cb8b67-485a-4c81-bd7b-e65a130d876d)